micrometer-observation/src/main/java/io/micrometer/observation/ObservationRegistry.java

```diff
@@ -139,7 +139,7 @@ public ObservationConfig observationFilter(ObservationFilter observationFilter)
          * @param observationConventions observation conventions
          * @return This configuration instance
          */
-        public ObservationConfig observationConvention(GlobalObservationConvention<?>... observationConventions) {
+        public ObservationConfig observationConventions(GlobalObservationConvention<?>... observationConventions) {
```

Comment:
If we are to have `observationConventions` then we should also have `observationPredicates`, `observationHandlers` etc.

Reply:
Those are singular right now:
```java
public ObservationConfig observationPredicate(ObservationPredicate predicate)
public ObservationConfig observationFilter(ObservationFilter observationFilter) 
```
while `observationConventions` has varargs:
```java
public ObservationConfig observationConventions(GlobalObservationConvention<?>... observationConventions)
```

So either:
```java
public ObservationConfig observationPredicates(ObservationPredicate... predicates)
public ObservationConfig observationFilters(ObservationFilter... observationFilters) 
public ObservationConfig observationConventions(GlobalObservationConvention<?>... observationConventions)
```
or:
```java
public ObservationConfig observationPredicate(ObservationPredicate predicate)
public ObservationConfig observationFilter(ObservationFilter observationFilter) 
public ObservationConfig observationConvention(GlobalObservationConvention<?> observationConvention)
```

What do you think?

Reply:
I checked the usages of this, even where we could use varargs we don't and call the method multiple times, I will move it to the singular version so that it will match with `observationPredicate` and `observationFilter`.